workflows/rnaseq/downstream/rnaseq.Rmd

```diff
@@ -29,7 +29,10 @@ output:
 ```
 
 ```{r global_options, include=FALSE}
-knitr::opts_chunk$set(warning=FALSE, message=FALSE)
+knitr::opts_chunk$set(warning=FALSE, message=FALSE, autodep=TRUE)
+
+#automatically identify cache dependencies
+knitr::dep_auto()
```

Comment:
whoa . . . have you had a chance to test this in practice?

Reading through https://yihui.name/knitr/demo/cache/ to learn more about other things in the cache I've missed, I saw https://github.com/yihui/knitr/issues/238 which uses the md5sum of an input data file. 

What are your thoughts on using the md5sum of the featurecounts file for this?

Reply:
It takes about .03 seconds for a 5 MB file, not a bad idea. 
Perhaps also the sampletable?
I have not had a chance to test this in practice using our data and variables. However, I have tested it with our code and dummy variables. I'll test it a little now to see how it fares.